src/NUnitFramework/nunitlite.tests/nunitlite.tests.csproj

```diff
@@ -1,58 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Common.props" />
   <PropertyGroup>
-    <AssemblyName>nunitlite.tests</AssemblyName>
-    <RootNamespace>NUnitLite.Tests</RootNamespace>
-    <OutputType>Exe</OutputType>
     <TargetFrameworks>net20;net35;net40;net45;netcoreapp1.1</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <DebugType Condition="'$(TargetFramework)' != '' AND '$(TargetFramework)' != 'netcoreapp1.1'">Full</DebugType>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
-    <DefineConstants>TRACE</DefineConstants>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>NUnitLite.Tests</RootNamespace>
+    <DebugType>Full</DebugType>
```

Comment:
This is the only place where we set `DebugType`. What is the default?

Reply:
In the new csproj the default is Portable.

Reply:
We also set this in nunit.framework.tests.csproj.